client/src/pages/Contest/NewContest.tsx

```diff
@@ -32,11 +35,23 @@ const NewContest: FunctionComponent = (): JSX.Element => {
       title: string;
       description: string;
       prizeAmount: string;
-      date: string;
+      deadline: Date | null;
       time: string;
       timezone: string;
     }>,
   ) => {
+    createContestAPI(title, description, prizeAmount, deadline).then((data) => {
+      if (data.error) {
+        setSubmitting(false);
```

Comment:
I don't think we to `setSubmitting` to false here and below, as it's being set outside the else/if construct (line 55)?

Reply:
removed setSubmitting(false) inside if/else condition.